config/localization/en/app.json

```diff
@@ -1707,7 +1707,7 @@
       "PROTOCOL_VAULT": "dYdX Protocol Vault",
       "LOCKED_BALANCE": "Locked Balance",
       "WITHDRAW_TOO_HIGH": "You cannot withdraw more than your vault balance.",
-      "WITHDRAW_TOO_LOW": "Your withdrawal amount is below the $20 minimum withdrawal threshold.",
+      "WITHDRAW_TOO_LOW": "Your withdrawal amount is below the $20 minimum withdrawal threshold. If your vault balance is below the minimum withdrawal threshold, you must withdraw the entire vault balance.",
```

Comment:
Since we are editing an existing key, make sure you audit new automated LILT pushes to see if it updates. If not you may have to reach out to the LILT team for manual intervention